src/tribler-core/tribler_core/components/metadata_store/db/store.py

```diff
@@ -136,14 +136,15 @@
 
 class MetadataStore:
     def __init__(
-        self,
-        db_filename: Union[Path, type(MEMORY_DB)],
-        channels_dir,
-        my_key,
-        disable_sync=False,
-        notifier=None,
-        check_tables=True,
-        db_version: int = CURRENT_DB_VERSION,
+            self,
```

Comment:
**Question:** My impression is that these changed lines should be indented with four spaces. Why are we adding an additional four spaces per line here?

Reply:
It is [black formatting](https://black.readthedocs.io/en/stable/) 🤷